### ISSUE TITLE
Can't read 0 as number

### ISSUE THREAD

**Issue by vedantroy** [lpgera/ts-env#12]
Here:
```
process.env.FOO = "0"
export const FOO = tsEnv.numberOrThrow("FOO")
```

**Comment by lpgera** [comment]
Nice find! It must have something to do with the throwing logic, as the simple `number()` reading function [works as intended](https://github.com/lpgera/ts-env/blob/356dfc4e74da2c66b900b9e76c9c43bf7ba87b9a/tests/number.test.ts#L13-L16).

I'll try to look into this later today and release a patch hopefully soon.

**Comment by lpgera** [comment]
@vedantroy the fix is published in version 1.2.5 🎉

Please let me know if you still face any issues.
